percy/percy.test.js

```diff
@@ -440,18 +366,70 @@ test('OTP-RR', async () => {
   await viewAllOptionsButton.click()
   await page.waitForTimeout(1000)
 
-  // Need to explicitly select the first itinerary to reset map position
-  await page.goto(`${page.url()}&ui_activeItinerary=1`)
+  // Select first itineary for printing
+  // FIXME: Navigation out of print view is funky.
+  await page.goto(`${page.url()}&ui_activeItinerary=0`)
   await page.waitForTimeout(2000)
 
-  if (!OTP_RR_PERCY_CALL_TAKER) {
-    await page.click('label[title="Bike"]')
-    await page.click('#plan-trip')
-    await page.waitForTimeout(5000)
+  await page.click('.button-container:nth-of-type(2)')
+  await page.waitForTimeout(500)
+  if (isMobile) {
+    // Printable itinerary screenshot on mobile only better page ration (and to save allowance).
+    await percySnapshotWithWait(page, 'Printable Itinerary')
   }
+}
 
-  await percySnapshotWithWait(
-    page,
-    'Batch Itinerary With Transit Showing Bikes'
-  )
+test('OTP-RR Mobile', async () => {
+  const page = await loadPath('/')
+  await page.setUserAgent('android')
+  await page.setViewport({
+    height: 1134,
+    width: 750
+  })
+  // Need to reload to load mobile view properly
+  await page.reload()
```

Comment:
This trick isn't working for me. I think the mobile tests need to be executed separately for them to register correctly in percy

Reply:
Not sure how Percy is reporting snapshots, but I have re-separated the tests with [c4fd913](https://github.com/opentripplanner/otp-react-redux/pull/972/commits/c4fd913c4c53bc752ec01444ed449d20bd5f580d).

Reply:
So far, I think there 18 snapshots taken total for each commit, but it is not obvious to list them all in Percy.